packages/main/src/utils/Strings.js

```diff
@@ -1,65 +1,86 @@
 /**
- * Escape a regular expression text so that it can be used in a regular expression.
- * @param {string} text The raw text
- * @returns The regular expression text, with special characters having been escaped.
- */
+	* Escape a regular expression text so that it can be used in a regular expression.
+	* @param {string} text The raw text
+	* @returns The regular expression text, with special characters having been escaped.
+	*/
 export function escapeRegex(text) {
-  return text.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+	return text.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
 }
 
 /**
- * Replace all instances of a particular find text.
- * @param {string} text The text to find/replace on
- * @param {string} find The string to find
- * @param {string|(s:string)=>string} replace The replace string or function
- * @param {boolean=} caseInsensitive true if the replace is case insensitive, false by default
- * @return {string} The string with all matches replaced.
- */
+	* Replace all instances of a particular find text.˝
+	* @param {string} text The text to find/replace on
+	* @param {string} find The string to find
+	* @param {string|function} replace The replace string or function
+	* @param {boolean=} caseInsensitive true if the replace is case insensitive, false by default
+	* @return {string} The string with all matches replaced.
+	*/
 export function replaceAll(text, find, replace, caseInsensitive) {
-  return text.replace(new RegExp(escapeRegex(find), `${caseInsensitive ? 'i' : ''}g`), replace);
+	return text.replace(new RegExp(escapeRegex(find), `${caseInsensitive ? "i" : ""}g`), replace);
 }
 
-// Cache the regular expressions for repeated use
-const markupReplace = `&=&amp,>=&gt;,<=&lt;,"=&quot;,'=&#39;`.split(",")
-  .map((p) => p.split('='))
-  .map(([f, r]) => [new RegExp(escapeRegex(f), 'g'), r]);
+// HTML needs to escape all attribute special characters, plus a few extras
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):** Let's use the rationale that https://en.wikipedia.org/wiki/Character_encodings_in_HTML#HTML_character_references specifies...

```suggestion
// HTML needs to escape all characters that are used to delimit markup
```